ImplementationGuide/markdown/Datenobjekte/Datenobjekte_Subscription.md

```diff
@@ -0,0 +1,3 @@
+## Subscription patient-merge  [(R5 Backport Subscription)](https://hl7.org/fhir/uv/subscriptions-backport/components.html)
```

Comment:
question: ggf. umbenennen in "patient merge notification" ?

Reply:
Wie bereits besprochen subscribe ich auf patient-merge. Und die Subscription gibt mir Bescheid sobald eine patient-merge stattfand.